packages/xmake/build.sh

```diff
@@ -2,8 +2,8 @@ TERMUX_PKG_HOMEPAGE=https://xmake.io/
 TERMUX_PKG_DESCRIPTION="A cross-platform build utility based on Lua"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Ruki Wang @waruqi"
-TERMUX_PKG_VERSION=2.5.6
+TERMUX_PKG_VERSION=2.5.7
 TERMUX_PKG_REVISION=2
```

Comment:
Kindly delete TERMUX_PKG_REVISION.  
While updating versions it bas to be removed.